### PR TITLE
Add HTTP 418

### PR DESCRIPTION
**Pull request by msieurtoph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ UNAUTHORIZED                        | 401   | Unauthorized
 UNPROCESSABLE_ENTITY                | 422   | Unprocessable Entity
 UNSUPPORTED_MEDIA_TYPE              | 415   | Unsupported Media Type
 USE_PROXY                           | 305   | Use Proxy
+I_TEAPOT                            | 418   | I'm a teapot
```
Comment (prettymuchbryce) [inline README.md]: Can we change the constant to `IM_A_TEAPOT`, and put it in alphabetical order with the others?

File: index.js [hunk 2]
```diff
@@ -65,11 +65,12 @@ statusCodes[exports.UNAUTHORIZED = 401] = "Unauthorized";
 statusCodes[exports.UNPROCESSABLE_ENTITY = 422] = "Unprocessable Entity";
 statusCodes[exports.UNSUPPORTED_MEDIA_TYPE = 415] = "Unsupported Media Type";
 statusCodes[exports.USE_PROXY = 305] = "Use Proxy";
+statusCodes[exports.I_TEAPOT = 418] = "I'm a teapot";
```
Comment (prettymuchbryce) [inline index.js]: In alphabetical order by constant name.

**Comment by msieurtoph** [comment]
Done! :)
I did not notice the alphabetical order.

**Comment by prettymuchbryce** [comment]
Thanks for the contribution. Will bump the version shortly.

**Pull request merged**